docs/tutorials/local_sine_tutorial.rst

```diff
@@ -302,8 +300,9 @@ circumstances where Python's multiprocessing does not. In this section, we'll
 explore modifying the above code to use MPI instead of multiprocessing.
 
 We recommend MPICH_ for this tutorial, which can be found for a variety of systems
-here_. You also need mpi4py, which can be downloaded via ``pip3 install mpi4py``.
-If this doesn't work, try appending ``--user`` to the end of the command.
+here_. You also need ``mpi4py``, which can be downloaded via ``MPICC=$(which mpicc) pip3 install mpi4py``.
```

Comment:
I'm thinking, though need to check to be certain, that this line would do the default anyway. I have seen it recommended like this before, but I think mpi4py will grab this compiler. However, they might want to specify something different. On Bebop you have to use mpiicc to get actual intel compiler. On a cray system it might just be cc. I've been trying to find a line that always works. But maybe we should just explain it. eg.

can be downloaded via ``pip3 install mpi4py``

or if you are using a different MPI ``MPICC=<path/to/mpi C compiler>  pip3 install mpi4py``

see: https://mpi4py.readthedocs.io/en/stable/install.html
or maybe we just say if you are using a different MPI compiler see this link.